packages/components/psammead-episode-list/src/__snapshots__/index.test.jsx.snap

```diff
@@ -515,14 +530,9 @@ exports[`Episode List  should render radio episodes correctly 1`] = `
           </span>
         </a>
         <span
-          class="emotion-16 emotion-17"
+          class="emotion-44 emotion-17"
           dir="ltr"
         >
-          <span
-            aria-hidden="true"
-            class="emotion-44 emotion-45"
-          />
-           
           21 octobre 2020
         </span>
```

Comment:
How come we aren't seeing the time HTML element here?

Reply:
The html element here wasn't for the time but for the separator. This is what we fixed. This span was only there for the vertical separator and  we've fixed that in this pr